CONTRIBUTING.md

```diff
@@ -26,29 +26,35 @@ Make sure you get a `:thumbsup`, `:+1`, or `LGTM` from another collaborator befo
 
 Make sure the relevant code has been reviewed and merged.
 
-### Developing inside a node_modules folder
+### Work on Multiple Packages / Repositories
 
-First make sure you are comfortable with [how require works](https://github.com/maxogden/art-of-node#how-require-works) in node.
+A very simple approach:
 
-We recommend creating a folder somewhere manually called `node_modules`. For example in `~/code/node_modules`. Clone all of your git copies of modules that you want to work on into here, so for example:
+Create a folder somewhere manually called `node_modules`. For example in `~/node_modules`. Clone all of your git copies of modules that you want to work on into here, so for example:
 
-- `~/code/node_modules/dat`
-- `~/code/node_modules/hyperdrive`
-
-When you run `npm install` inside of `~/code/node_modules/dat`, dat will get its own copy of `hyperdrive` (one if its dependencies) inside `~/code/node_modules/dat/node_modules`. However, if you encounter a bug in hyperdrive that you need to fix, but you want to test your fix in dat, you want dat to use your git copy of hyperdrive at `~/code/node_modules/hyperdrive` and not the npm copy of hyperdrive at `~/code/node_modules/dat/node_modules/hyperdrive`.
+```sh
+mkdir node_modules
+cd node modules
+git clone https://github.com/MetaMask/metamask-extension
+git clone https://github.com/MetaMask/provider-engine
+# clone more repos...
```

Comment:
Might also be worth noting that you should `npm install` in each of those projects.